src/pages/HomePage.jsx

```diff
@@ -0,0 +1,282 @@
+      <Box flex="1">
+        <AlertTitle>{title}</AlertTitle>
+        <AlertDescription display="block">{description}</AlertDescription>
+        {closeButton && (
+          <CloseButton position="absolute" right="8px" top="8px" onClick={() => setClosed(true)} />
+        )}
+        {goToLogButton && (
+          <Button float="right" colorScheme="red" size="sm" rightIcon={<ArrowForwardIcon />}>
+            Go to Log
+          </Button>
+        )}
+      </Box>
+    </Alert>
+  );
+};
+
+const SegAssignment = props => {
+  const { title, place, description } = props;
+  return (
+    <Box w="400px">
+      <Text fontSize="16px" color="#231F20">
+        {title}
+      </Text>
+      <Text fontSize="16px" color="#4A5568">
+        {place}
+      </Text>
+      <FiMapPin color="#156071" style={{ display: 'inline' }} />
+      <Text marginLeft="3" as="u" fontSize="16px" color="#156071">
+        <Link href="https://www.google.com/maps" isExternal>
+          Google Maps Link
+        </Link>
+      </Text>
+      <Text />
+      <FiImage color="#156071" style={{ display: 'inline' }} />
+      <Text marginLeft="3" as="u" fontSize="16px" color="#156071">
+        <Link href="https://www.google.com/maps" isExternal>
+          Map Image
+        </Link>
+      </Text>
+      <Text fontSize="16px" color="#4A5568">
+        {description}
+      </Text>
+    </Box>
+  );
+};
+
+const UnsubmittedLogDraft = props => {
+  const { title, timeDescription } = props;
+  return (
+    <Box align="center" border="2px" borderRadius="md" borderColor="lightgray" w="400px" h="125px">
+      <Box w="400px" h="15px" />
+      <Text fontSize="16px" pl="6" textAlign="left">
+        {title}
+      </Text>
+      <Text fontSize="16px" pl="6" textAlign="left" color="#4A5568">
+        {timeDescription}
+      </Text>
+      <Box w="400px" h="10px" />
+      <Button w="350px" bgColor="#2BC0E3" size="sm" rightIcon={<ArrowForwardIcon />}>
+        Go to Log
+      </Button>
+    </Box>
+  );
+};
+
+const RecentlySubmittedLog = props => {
+  const {
+    title,
+    timeDescription,
+    badgeColor,
+    badgeDescription,
+    borderColor,
+    goToLogButton,
+    marginAmt,
+  } = props;
+  return (
+    <Box
+      align="center"
+      border="2px"
+      borderRadius="md"
+      borderColor={borderColor}
+      w="400px"
+      h="155px"
+    >
+      <Box w="400px" h="15px" />
+      <Text fontSize="16px" pl="6" textAlign="left">
+        {title}
+      </Text>
+      <Text fontSize="16px" pl="6" textAlign="left" color="#4A5568">
+        {timeDescription}
+      </Text>
+      <Badge marginRight={marginAmt} bg={badgeColor} textColor="white">
+        {badgeDescription}
+      </Badge>
+      <Box w="400px" h="15px" />
+      {goToLogButton && (
+        <Button w="350px" bgColor="#2BC0E3" size="sm" rightIcon={<ArrowForwardIcon />}>
+          Go to Log
+        </Button>
+      )}
+    </Box>
+  );
+};
+
+// GET BACKEND DATA
+const HomePage = props => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'props' is defined but never used.